src/storage/src/source/postgres/replication.rs

```diff
@@ -467,7 +489,13 @@ pub(crate) fn render<G: Scope<Timestamp = MzOffset>>(
                 if will_yield {
                     upper_cap_set.downgrade([&data_upper]);
                     data_cap_set.downgrade([&data_upper]);
+                    trace!(
+                        %id,
+                        "timely-{worker_id} yielding at lsn={}",
+                        data_upper
+                    );
                     rewinds.retain(|_, (_, req)| data_cap_set[0].time() <= &req.snapshot_lsn);
+                    trace!(%id, "timely-{worker_id} remaining rewinds {rewinds:?}");
```

Comment:
Can we combine these traces in a single one?

Reply:
ah yes sorry this was left from debugging